6.7.3/centos/Dockerfile

```diff
@@ -9,7 +9,7 @@
 # For further information, please contact Curity AB.
 #
 
-FROM centos:centos8 AS openssl-builder
+FROM quay.io/centos/centos:stream8 AS openssl-builder
```

Comment:
This is official?

Reply:
yes, got it from here: https://www.centos.org/centos-stream/

Reply:
Great. This looks good in that case.